plugins/pa11y/src/tasks/pa11y.ts

```diff
@@ -0,0 +1,42 @@
+const pa11y = require('pa11y') // eslint-disable-line
+import { readState } from '@dotcom-tool-kit/state'
+import { Task } from '@dotcom-tool-kit/types'
+import { ToolKitError } from '@dotcom-tool-kit/error'
+import { Pa11ySchema } from '@dotcom-tool-kit/types/lib/schema/pa11y'
+
+export default class Pa11y extends Task<typeof Pa11ySchema> {
+  async run(): Promise<void> {
+    const reviewState = readState('review')
+    // if we've built a review app, test against that, not the app in the config
+    if (reviewState) {
+      this.options.host = `https://${reviewState.appName}.com`
+    }
+
+    const results = await pa11y(this.options.host ? this.options.host : '')
```

Comment:
**suggestion**: perhaps we should throw a toolkit error if this.options.host is undefined/null instead of passing in a empty string, to be more explicit